locales/de-de.json

```diff
@@ -0,0 +1,74 @@
+{
+	"select-user-type.who_you_are": "Was du bist? (Wähle jetzt aus, du kannst es später ändern)",
+	"select-user-type.driver": "Fahrer",
+	"select-user-type.passenger": "Passagier",
+	"request-phone.type_your_phone": "Bitte sende uns deine Telefonnummer. Wir geben diese Nummer nicht ohne Erlaubnis weiter.",
+	"request-phone.button_text": "Nummer senden",
+	"request-phone.all_set": "👌 Alles klar soweit, du kannst jetzt eine Fahrt bestellen!",
```

Comment:
What do you think about:

Das war alles, du kannst jetzt deine erste Fahrt bestellen!